website/pages/en/network/delegating.mdx

```diff
@@ -2,13 +2,15 @@
 title: Delegating
 ---
 
-Delegators are network participants who delegate (i.e., "stake") GRT to one or more Indexers. Delegators contribute to securing the network without running a Graph Node themselves.
+Delegators are network participants who delegate (i.e., "stake") GRT to one or more Indexers. Delegators help secure the network without running a Graph Node themselves.
 
-By delegating to an Indexer, Delegators earn a portion of the Indexer's query fees and rewards. The amount of queries an Indexer can process depends on the Indexer's own (and delegated) stake and the price the Indexer charges for each query, so the more stake that is allocated to an Indexer, the more potential queries they can process.
+They earn a portion of the Indexer's query fees and rewards by delegating to an Indexer. The amount of queries an Indexer can process depends on the Indexer's own stake, the delegated stake, and the price the Indexer charges for each query. Therefore, the more stake that is allocated to an Indexer, the more potential queries they can process.
```

Comment:
```suggestion
Delegators earn a portion of the Indexer's query fees and rewards by delegating to an Indexer. The amount of queries an Indexer can process depends on the Indexer's own stake, the delegated stake, and the price the Indexer charges for each query. Therefore, the more stake that is allocated to an Indexer, the more potential queries they can process.
```
And maybe the second sentence in the previous paragraph can start with "They" instead of "Delegators"?